R/uti-CombineSpecs.R

```diff
@@ -0,0 +1,25 @@
+#' Combine Specifications
+#'
+#' This function combines a list of specifications into a single specification.
+#'
+#' @param lSpecs A list of specifications.
+#' @return A list representing the combined specification.
+#' @examples
+#' all_wf <- gsm::MakeWorkflowList()
+#' all_specs <- CombineSpecs(all_wf)
+#'
+#' @export
+
+CombineSpecs <- function(lSpecs){
+    all_specs <- list()
+    for (spec in lSpecs){
+        for (domain in names(spec)){
+            if (domain %in% names(all_specs)){
+                all_specs[[domain]] <- c(all_specs[[domain]], spec[[domain]])
+            } else {
+                all_specs[[domain]] <- spec[[domain]]
+            }
```

Comment:
```suggestion
            all_specs[[domain]] <- c(all_specs[[domain]], spec[[domain]])
```
The if/else is redundant. `all_specs[["new_domain"]]` is `NULL`, and `c(NULL, spec[["new_domain"]]` is `spec[["new_domain"]]`, so they can combine this way in either case.